utils/crud/mock_store_test.go

```diff
@@ -0,0 +1,19 @@
+package crud
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMockStore(t *testing.T) {
```

Comment:
I think it's the first time I saw a mock tested 😸

Reply:
This test has been around since the beginning I think? TRUST NO ONE (including our mocks) 😉